### PR TITLE
Problem: (Fix #972) Missing MultiSig pubkey methods in ClientRPC and ClientCLI

### LINKED ISSUE #972: Problem: Create and list MultiSIg public keys is missing in ClientCLI

### PR DESCRIPTION
**Pull request by calvinlauco**
Solution: Add new and list methods to ClientRPC and ClientCLI

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,9 +9,9 @@ __pycache__
 *.egg-info
 
 # Storage
+/.storage*
```
Comment (yihuang) [inline .gitignore]: superseds the `/.storage`

Comment (calvinlauco) [reply .gitignore]: Removed `/.storage`, thanks!

File: client-rpc/src/rpc/multisig_rpc.rs [hunk 2]
```diff
@@ -5,12 +5,19 @@ use jsonrpc_derive::rpc;
...
 use client_core::{MultiSigWalletClient, WalletClient};
 
 use crate::server::{to_rpc_error, WalletRequest};
 
 #[rpc]
 pub trait MultiSigRpc: Send + Sync {
+    #[rpc(name = "multiSig_newAddressPublicKey")]
+    fn new_address_public_key(&self, request: WalletRequest) -> Result<String>;
+
+    #[rpc(name = "multiSig_listAddressPublicKeys")]
```
Comment (yihuang) [inline client-rpc/src/rpc/multisig_rpc.rs]: The RPC endpoint name change is breaking, the `test_watch_only_wallet.py` and `chainrpc.py` needs to change accordingly.

**Comment by tomtau** [comment]
bors try

**Comment by tomtau** [comment]
and perhaps add an entry in CHANGELOG, as this is a breaking change

**Comment by tomtau** [comment]
bors try

**Comment by calvinaco** [comment]
bors retry

**Pull request merged**